### PR TITLE
[pm-api] Add example packages for testing

### PR DESCRIPTION
**Pull request by njgheorghita**
branched from #949 

### What was wrong?
Improved tests
Added an integration test with `StandardToken`
Add packages folder to store packages for testing

#### Cute Animal Picture
image

### CODE REVIEW

File: tests/core/pm-module/packages/owned/1.0.0.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    "description": "Reusable contracts which implement a privileged 'owner' model for authorization.",
+    "keywords": [
+      "authorization"
+    ],
+    "links": {
+      "documentation": "ipfs://QmUYcVzTfSwJoigggMxeo2g5STWAgJdisQsqcXHws7b1FW"
+    }
+  },
+  "sources": {
+    "./contracts/Owned.sol": "ipfs://Qme4otpS88NV8yQi8TfTP89EsQC5bko3F5N1yhRoi6cwGV"
```
Comment (pipermerriam) [inline tests/core/pm-module/packages/owned/1.0.0.json]: Should these be represented in their *compact* form.  I don't think I've seen a PR in ethpm to enforce that data format, so it's probably something that needs to have an issue created for, but I suspect we'll run into an error in this repository at some point in the future if we use non-canonical representations of the manifests here.

Comment (njgheorghita) [reply tests/core/pm-module/packages/owned/1.0.0.json]: Good point, I'll add that issue to `py-ethpm`. And once it gets implemented there it'll force us to update the contracts here.

Comment (davesque) [reply tests/core/pm-module/packages/owned/1.0.0.json]: Should I understand "compact" to mean `ipfs:Qme...`?

Comment (njgheorghita) [reply tests/core/pm-module/packages/owned/1.0.0.json]: @davesque I don't think it has anything to do with the contents of the uri - just to enforce the guidelines [here in ethpm-spec](http://ethpm.github.io/ethpm-spec/package-spec.html#document-format)

**Comment by carver** [comment]
If anyone is waiting on me, please go ahead and merge this and other PRs into the `pm-abi` branch at will :+1:

**Pull request merged**